sdk/servicebus/azure-servicebus/azure/servicebus/amqp/_amqp_message.py

```diff
@@ -201,7 +220,17 @@ def __repr__(self):
         return "AmqpAnnotatedMessage({})".format(message_repr)[:1024]
 
     def _from_amqp_message(self, message):
-        # populate the properties from an uamqp message
+        # populate the properties from an pyamqp message
+        if message[5]:
```

Comment:
out of curiosity: is this more performant than calling message.data?

Reply:
Generally yes - accessing a tuple by index is the tiniest bit faster.